src/components/ebay-pill/index.js

```diff
@@ -19,10 +18,9 @@ module.exports = require('marko-widgets').defineComponent({
         }
     },
     handleButtonClick(event) {
-        this.setState('pressed', !this.state.pressed);
-        emitAndFire(this, 'button-click', event);
-    },
-    handleButtonEscape(event) {
```

Comment:
Why are we removing this? Just curious.

Reply:
We aren't using the `<ebay-button>` anymore. That event we had to manually bubble, but we don't need to anymore. Note the changes in the template removing the `button-escape` event because we changed to a native button.